Runigram.java

```diff
@@ -112,7 +164,18 @@ public static Color[][] scaled(Color[][] image, int width, int height) {
 	 */
 	public static Color blend(Color c1, Color c2, double alpha) {
 		//// Replace the following statement with your code
-		return null;
+		int red1,green1,blue1,red2,green2,blue2,newRed,newGreen,newBlue;

```

Comment:
add spaces int red1, green1, blue1;
Also there's no need to declare the variables without initialization,
It would be better:
int red1 = c1.getRed();
int green1 = c1.getGree();